MIGRATION.md

```diff
@@ -1,98 +1,5 @@
 # Migration
 
-## Table of contents
```

Comment:
If I'm not mistaken, this file can be viewed only within Github, there is no related page on the website. So I'd keep the TOC here.

Reply:
ah, good point